client/apps/hello_world/store/configure_store.js

```diff
@@ -8,5 +8,12 @@ const middleware = [API];
 // make the modifications in this file by extending the configureStore
 // or copy pasting the code into this file.
 export default function(initialState) {
-  return configureStore(initialState, rootReducer, middleware);
+  const store = configureStore(initialState, rootReducer, middleware);
+  if (module.hot) {
+    module.hot.accept('../reducers', () => {
+      const nextRootReducer = require("../reducers/index").default;
+      store.replaceReducer(nextRootReducer);
+    });
+  }
+  return store;
 }
```

Comment:
When you dont use redux you can start hot reload in one place at the very root of the app. But with redux you cant hot reload store creation (the provider component) so we start hot reloading again at the reducer lever so the only thing that wont hot reload is your store. Which is good because it means your data will still exist in your store after a hot reload.